policyengine_us/parameters/gov/states/wv/tax/income/subtractions/other_retirement/amount.yaml

```diff
@@ -0,0 +1,15 @@
+description: Combined amounts of retirement income reported on the federal return received from The West Virginia Teachers' Retirement System 
+             and/or The West Virginia Public Employees' Retirement System and from Federal Retirement must not exceed this amount.
+             
+values:
+  2022-01-01: 2_000
```

Comment:
add 2021 values from 2021 tax form + add source to references

Reply:
2021 values are the same as 2022 values.